upgrade/1.0.11/Stage_0_Prerequisites.md

```diff
@@ -0,0 +1,181 @@
+# Stage 0 - Prerequisites and Preflight Checks
+
+> **NOTE:** CSM-1.0.1 or later is required in order to upgrade to CSM-1.0.11.
+>
+> **NOTE:** Installed CSM versions may be listed from the product catalog using the following command. This will sort a semantic version without a hyphenated suffix after the same semantic version with a hyphenated suffix, e.g. 1.0.11 > 1.0.11-beta.19.
+>
+
+Use the following command can be used to check the CSM version on the system:
+
+```bash
+ncn# kubectl get cm -n services cray-product-catalog -o json | jq -r '.data.csm'
+```
+
+This check will also be conducted in the 'prerequisites.sh' script listed below and will fail if the system is not running CSM-1.0.1 or CSM-1.0.10.
+
+>**`IMPORTANT:`**
+> 
+> Before running any upgrade scripts, be sure the Cray CLI output format is reset to default by running the following command:
+>
+>```bash
+> ncn# unset CRAY_FORMAT
+>```
+
+## Stage 0.1 - Install latest docs RPM
+
+1. Install latest document RPM package:
+
+    * Internet Connected
+
+        ```bash
+        ncn-m001# cd /root/
+        ncn-m001# wget https://storage.googleapis.com/csm-release-public/shasta-1.5/docs-csm/docs-csm-latest.noarch.rpm
+        ncn-m001# rpm -Uvh docs-csm-latest.noarch.rpm
+        ```
+
+    * Air Gapped
+
+        ```bash
+        ncn-m001# cd /root/
+        ncn-m001# rpm -Uvh [PATH_TO_docs-csm-*.noarch.rpm]
+        ```
+
+## Stage 0.2 - Update `customizations.yaml`
+
+Perform these steps to update `customizations.yaml`:
+
+1. Extract `customizations.yaml` from the `site-init` secret:
+
+    ```bash
+    ncn-m001# cd /tmp
+    ncn-m001# kubectl -n loftsman get secret site-init -o jsonpath='{.data.customizations\.yaml}' | base64 -d - > customizations.yaml
+    ```
+
+1. Update `customizations.yaml`:
+
+    ```bash
+    ncn-m001# /usr/share/doc/csm/upgrade/1.0.11/scripts/upgrade/update-customizations.sh -i customizations.yaml
+    ```
+
+1. Update the `site-init` secret:
+
+    ```bash
+    ncn-m001# kubectl delete secret -n loftsman site-init
+    ncn-m001# kubectl create secret -n loftsman generic site-init --from-file=customizations.yaml
+    ```
+
+1. If [using an external Git repository for managing customizations](../../install/prepare_site_init.md#version-control-site-init-files) as recommended,
```

Comment:
If I was using an external git repository to manage my customizations.yaml I'd do this to get the existing customizations.yaml rather than getting it from the site-init secret.

Reply:
i believe this step is instructing them to keep their copy in vcs current with what's in the secret (if they are doing that sort of thing)